### PR TITLE
Add codes.loers.Punchclock

### PR DESCRIPTION
**Pull request by floers**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: codes.loers.Punchclock.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: codes.loers.Punchclock
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
```
Comment (barthalion) [inline codes.loers.Punchclock.yml:11]:
```suggestion
  - --socket=fallback-x11
```

**Comment by barthalion** [comment]
bot, build codes.loers.Punchclock

**Comment by barthalion** [comment]
bot, build codes.loers.Punchclock

**Comment by barthalion** [comment]
```
builddir/files/share/appdata/codes.loers.Punchclock.appdata.xml: FAILED:
• tag-missing           : <url> is not present
```

File: codes.loers.Punchclock.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+app-id: codes.loers.Punchclock
+runtime: org.gnome.Platform
+runtime-version: "42"
```
Comment (hfiguiere) [inline codes.loers.Punchclock.yml:3]: gnome 43 is avaiable.

File: codes.loers.Punchclock.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+app-id: codes.loers.Punchclock
```
Comment (hfiguiere) [inline codes.loers.Punchclock.yml:1]: `loers.code` is a parked domain, is this controlled by the developers?

Comment (floers) [reply codes.loers.Punchclock.yml:1]: yes. It's my domain. I have it for exactly this purpose

File: codes.loers.Punchclock.yml (hunk 2 above)
Comment (floers) [reply codes.loers.Punchclock.yml]: I updated it to 43

File: codes.loers.Punchclock.yml (hunk 1 above)
Comment (floers) [reply codes.loers.Punchclock.yml]: I removed --socket=x11

**Comment by floers** [comment]
Hey, @hfiguiere, Thanks for tending with my request and giving feedback. I think I fixed everything.

**Comment by hfiguiere** [comment]
bot, build codes.loers.Punchclock

**Comment by floers** [comment]
Thanks :)

**Pull request closed without merging**